src/ngx_http_lua_ssl.h

```diff
@@ -25,6 +25,7 @@ typedef struct {
     ngx_str_t                session_id;
 
     int                      exit_code;  /* exit code for openssl's
+                                            set_client_hello_cb or
```

Comment:
This may introduce conflict between cert_by_lua and client_hello_by_lua? Is it possible I have both configured in the same server block?
@zhuizhuhaomeng  what do you think?

Reply:
this var has been shared by ssl_session_store_by, ssl_session_fetch_by, ssl_certificate_by. It only makes sense at one the same phase, So can be shared in multiple phases.